openapi/src/parts/controlled_azure_relay_namespace.yaml

```diff
@@ -20,6 +20,24 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/relayNamespace/create-result/{jobId}:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/JobId'
+    get:
+      summary: Retrieve information about an Azure Vm create job.
```

Comment:
⛏️ There are a few allusions in the swagger changes to creating a VM, but we're creating a relay namespace right?